tests/_client.lua

```diff
@@ -136,10 +141,20 @@ return function(class, title, sn_rules, callback, resize_increment, args)
     if args.maximize_after then
         options = options .. "maximize_after,"
     end
+    if args.resize then
+        options = table.concat {
+            options,
+            "resize_after_x=",
+            args.resize.x, ",",
+            "resize_after_y=",
+            args.resize.y, ","
+        }
+    end
```

Comment:
Can these values pleased be called `width` and `height` instead of `x` and `y`? (Also in the following commits)